symfony/monolog-bundle/3.7/config/packages/monolog.yaml

```diff
@@ -0,0 +1,62 @@
+# As of Symfony 5.1, deprecations are logged in the dedicated "deprecation" channel when it exists
```

Comment:
I would remove the "as of symfony 5.1" part

Reply:
reformatted a bit, this should avoid merge conflicts on our side :angel: